scripts/monero.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+# Run the right version of monero.bt depending on linux headers
+# https://github.com/cryptnono/cryptnono/pull/38
+
+from pathlib import Path
+from os import execl
+
+fpu_api = list(Path("/").glob("usr/src/*/arch/*/include/asm/fpu/api.h"))
```

Comment:
Nice!

Can you leave a comment on what this glob will match / is looking for? You can merge once that's in place :)

Reply:
Added, thanks!